azerothcore/mod-1v1-arena
# feat. Misc fixs

## Changes Proposed:
- Update the module, to correct an error, when scoring sands.

## Issues Addressed:

- Closes

## SOURCE:


## Tests Performed:

- Windows 10
- In-game

## How to Test the Changes:


1. Create a character
2. Interact with the NPC to perform different actions (create arena team, score, cancel, check stats, leave the team, etc.).
